app/models/wiky.rb

```diff
@@ -1,3 +1,6 @@
 class Wiky < ActiveRecord::Base
   belongs_to :user
+  validates :title, length: { minimum: 5 }
+  scope :public_wikys, -> { where(public: true) }
+  scope :private_wikys, -> {where(public: false)}
```

Comment:
Good use of scopes, well done. Consider renaming from `:public_wikys` to `:public` and `:private_wikys` to `:private`.